src/pages/registration/components/Organisms/RegistrationForm.jsx

```diff
@@ -0,0 +1,21 @@
+import { RectangleButton } from '../../../../components/Atoms/RectangleButton/RectangleButton.jsx';
+import { RegistrationInputs } from '../Molecules/RegistrationInputs.jsx';
+import { RegistrationLink } from '../Atoms/RegistrationLink.jsx';
+import css from './RegistrationForm.module.css';
+
+const RegistrationForm = () => {
+  return (
+    <div className={css.RegistrationForm}>
+      <div className={css.RegistrationModal}>
+        <h2 className={css.RegistrationTitle}>Registration</h2>
+        <RegistrationInputs />
+        <div className={css.RegistrationButton}>
+          <RectangleButton title="Sign Up" size="large" />
+        </div>
+      </div>
+      <RegistrationLink />
+    </div>
```

Comment:
missing `form` html tag ❌ 
add `method` and `action` attribute 💡

Reply:
Added the same logic in the SignInForm.jsx

Reply:
ok ✅